### DISCUSSION TITLE
v4.13.1 pseudo-real-time updates

### DISCUSSION THREAD

**Discussion by dsiminiuk** [floccusaddon/floccus#1141] [General]
I am not sure when this happened but I noticed that this version of floccus merges updates with NC bookmarks mere moments after a change in my browser.

This is exactly what I've been looking for in a bookmark sync solution since I stopped using Xmarks.

This is outstanding work Marcel. Expect another donation shortly.

Thank you!

**Comment by marcelklehr** [comment]
Hi @dsiminiuk

Yeah, I decreased the timeout a bit some time ago. Thanks for the kind words and your contribution. :hearts:

**Reply by marcelklehr** [reply]
@all-contributors Please add @dsiminiuk as a contributor for testing and ideas

**Reply by marcelklehr** [reply]
It turns out the behavior you're experiencing is a bit overzealous, it was set to 1,2s but should have been 7s :sweat_smile:

**Comment by dsiminiuk** [comment]
@marcelklehr Agreed, I have to make simple bookmark changes and wait for it to finish before making another. Is the delay configurable in the plugin or only the source?
